en/deploy/README.md

```diff
@@ -1,6 +1,6 @@
 # Deploy!
 
-> **Note**: The following chapter can be sometimes a bit hard to get through. Persist and finish it; deployment is an important part of the website development process. This chapter is placed in the middle of the tutorial so that your mentor can help with the slightly tricker process of getting your website online. This means you can still finish the tutorial on your own if you run out of time.
+> **Note** The following chapter can be sometimes a bit hard to get through. Persist and finish it; deployment is an important part of the website development process. This chapter is placed in the middle of the tutorial so that your mentor can help with the slightly tricker process of getting your website online. This means you can still finish the tutorial on your own if you run out of time.
```

Comment:
No, if you look at Gitbook, you'll see extra colons all over the place. And the ones that don't have it get rendered correctly.